modules/core/src/main/scala/sangria/execution/Executor.scala

```diff
@@ -29,107 +29,116 @@ case class Executor[Ctx, Root](
       operationName: Option[String] = None,
       variables: Input = emptyMapVars
   )(implicit um: InputUnmarshaller[Input]): Future[PreparedQuery[Ctx, Root, Input]] = {
-    val (violations, validationTiming) =
```

Comment:
this is mostly just moving things around so `queryValidator.validateQuery` has access to `unmarshalledVariables`